modules/nodes-dashboard-using-queries.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-dashboard-using.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-dashboard-using-queries_{context}"]
+= Customizing dashboard queries
+
+You can customize the default queries used to build the node metrics dashboard.
+
+.Procedure
+
+. Choose a metric and click *Inspect* to drill into the data. On this page, you will see the metric in detail, including an expanded visualization of the results of the query, the Prometheus query used to analyze the data, and the data subset used in the query.
+. Make any needed changes to the query parameters. 
```

Comment:
"Needed" I believe is allowed (unlike "need to"), however a light suggestion for "required", if you like:
```suggestion
. Make any required changes to the query parameters. 
```